backend/aws_helpers/dynamo_db_utils/trainspace_db.py

```diff
@@ -74,7 +74,7 @@ def updateStatus(trainspace_id: str, status: str, entryData: dict = None) -> boo
     """
     if entryData is None:
         entryData = {}
-    entryData["status"] = status
+    # entryData["status"] = status
```

Comment:
same question here

Reply:
Also to avoid dynamo enum issues I think